ilastik/applets/edgeTrainingWithMulticut/edgeTrainingWithMulticutGui.py

```diff
@@ -32,7 +32,24 @@ def initAppletDrawerUi(self):
         multicut_box = QGroupBox( "Multicut", parent=self )
         multicut_box.setLayout(multicut_layout)
         multicut_box.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Fixed)
-        
+        multicut_box.setEnabled(False)
+
+        # Enable multicut only when all edgeTraining->multicut connections
+        # have non-None inputs.
+        op = self.topLevelOperatorView
+        conns = (
+            op.Superpixels,
+            op.Rag,
+            op.EdgeProbabilities,
+            op.EdgeProbabilitiesDict,
+        )
+        def set_multicut_enabled(*args, **kwargs):
+            all_conns_ok = all(conn.value is not None for conn in conns)
```

Comment:
`conn.ready()` instead of `conn.value is not None` might be a better option in order to avoid `SlotNotReady` errors.
Or even: `conn.ready() and conn.value is not None`